src/bench.c

```diff
@@ -63,6 +63,10 @@ static void help(int default_iters) {
     printf("    ellswift_ecdh     : ECDH on ElligatorSwift keys\n");
 #endif
 
+#ifdef ENABLE_MODULE_SILENTPAYMENTS
+    printf("    silentpayments    : Silent payments recipient scanning\n");
```

Comment:
Not just scanning, right?

Reply:
Currently, I only have a benchmark for the scanning function (since that's the only function were performance is absolutely critical). Perhaps I can update this to `silentpayments_scanning` ?

Reply:
> I only have a benchmark for the scanning function

Ah, nevermind, I got this wrong.